src/DynamoCoreWpf/ViewModels/Search/NodeAutoCompleteSearchViewModel.cs

```diff
@@ -51,9 +53,17 @@ private void InitializeDefaultAutoCompleteCandidates()
         
         internal void PopulateAutoCompleteCandidates()
         {
+            searchElementsCache.Clear();
+
             if (PortViewModel == null) return;
 
             var searchElements = GetMatchingSearchElements();
+
+            foreach (NodeSearchElement element in searchElements)
+            {
+                searchElementsCache.Add(element);
+            }
```

Comment:
Instead of hard copy here, seems we can just do `searchElementsCache = GetMatchingSearchElements();` above?